config/bootstrap.php

```diff
@@ -64,6 +64,20 @@
 use Cake\Utility\Inflector;
 use Cake\Utility\Security;
 
+/**
+ * Read .env file if APP_NAME is not set.
+ *
+ * You can remove this block if you do not want to use environment
+ * variables for configuration when deploying.
```

Comment:
The comment is misleading. The file is. It meant for deploying but is to be used during development.

I understand the intent of your phrase, but it should be made clear what it is actually for.